packages/turf-clusters-dbscan/index.ts

```diff
@@ -2,7 +2,7 @@ import { GeoJsonProperties, FeatureCollection, Point } from "geojson";
 import { clone } from "@turf/clone";
 import { distance } from "@turf/distance";
 import { degreesToRadians, lengthToDegrees, Units } from "@turf/helpers";
-import { rbush as RBush } from "./lib/rbush-export.js";
+import rbush from "rbush";
```

Comment:
Its nice that we can get rid of the import hacks, I'm still trying to fix the underlying typings still.